src/Microsoft.Management.Configuration/Microsoft.Management.Configuration.idl

```diff
@@ -541,6 +556,13 @@ namespace Microsoft.Management.Configuration
         IGetAllSettingsResult GetAllSettings();
     }
 
+    [contract(Microsoft.Management.Configuration.Contract, 4)]
+    interface IGetAllUnitsConfigurationUnitProcessor requires IConfigurationUnitProcessor

```

Comment:
Do this interface and above IGetAllUnitsResult need to be registered in appxmanifest?